bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -354,6 +420,13 @@
 		<category>GarageDoor</category>
 		<state readOnly="true"/>
 	</channel-type>
+	<channel-type id="heatingTitle">
+		<item-type>String</item-type>
+		<label>Heating Title</label>
```

Comment:
Would this be better?
```suggestion
		<label>Heating Name</label>
```

Reply:
Please not ;-)
Do you still remember the discussion about Name vs. Title in https://github.com/openhab/openhab-addons/pull/16114?

Reply:
I forgot, but now i remember. Let's not change this. Don't think this is going to be the last time this title vs name is going to be talked about.